Tesla.css

```diff
@@ -0,0 +1,214 @@
+* {
+    margin: 0;
+    padding: 0;
+  }
+  
+  html,
+  body {
+    height: 100%;
+    width: 100%;
+  }
+  .body {
+    font-family: Montserrat;
+    font-weight: 600;
+    font-style: normal;
+    margin: 0;
+}
+.header {
+    position: relative;
+    font-family: Montserrat;
+    font-weight: 600;
+    font-style: normal;
+}
+.logo {
+    width: 166px;
+    height: 51px;
+    margin-left: 42.72px;
+    margin-top: 24px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla-n/pullRequest?prid=5749910)